server/app.js

```diff
@@ -29,7 +30,12 @@ app.post('/upload', (req, res, next) => {
       file: `public/${filename}`
     });
   });
+})
 
+app.post('/faceswap', (req, res, next) => {
+  const {divPositioning} = req.body;
+  const {top, left, height, width} = divPositioning;
+  shell.exec(`convert public/image.jpg -crop ${width}x${height}+${top}+${left} public/crop.jpg`)
```

Comment:
There's just a small mistake:

```suggestion
  shell.exec(`convert public/image.jpg -crop ${width}x${height}+${left}+${top} public/crop.jpg`)
```

I'll fix that with a commit!

Reply:
I'll fix that issue and then I'll merge your changes in 😄

Reply:
Thanks for catching and fixing that!